test/service/test_response.py

```diff
@@ -1290,6 +1290,18 @@ def test_file_response(self):
         file = one(one(keyword_response['hits'])['files'])
         self.assertElasticsearchResultsEqual(file, expected_file)
 
+    def test_faulty_filter(self):
+        url = self.base_url + "/index/samples"
+        params = self._params(size=10,
+                              filters={'notAFilter': {'is': 'foo'}})
+        response = requests.get(url, params=params)
+        self.assertEqual(response.status_code, 400)
+        expected = {
+            'Code': 'BadRequestError',
+            'Message': 'BadRequestError: Unknown facet `notAFilter`'
+        }
+        self.assertEqual(response.json(), expected)
```

Comment:
```suggestion
        self.assertEqual(expected, response.json())
```
Hannes prefers the first argument to be the expected value (reference: https://github.com/DataBiosphere/hca-metadata-api/pull/46#discussion_r619439494)

Reply:
@hannes-ucsc's choice is unintuitive to me since it goes against the common pattern for if statements

    if variable > 0:

and comparisons in general.

Even the unit test documentation seems to follow yours and my intuition here https://docs.python.org/3/library/unittest.html#organizing-test-code.

I'll change this since Hannes was very clear, but I hope he reconsiders. I scanned through our codebase and it looks like both kinds of comparisons are used more or less equally often.